lbu_files/nemeth.ctb

```diff
@@ -73,10 +123,13 @@ always @ 4-1
 always \\ 456-16
 always | 1256
 always $ 4-234
+#always \x27E8 46-46-12356 left angle bracket #Removed in 2023
+#always \x27E9 46-46-23456 right angle bracket #Removed in 2023
```

Comment:
Why did you add these rules but comment them out? Just remove them if they are not needed or add a comment why you left them

Reply:
Chris, I doed a first cleanup after consulted @NorbertMarkus the lbu_files/nemeth.ctb and lbu_files/nemetth_edit_with_switch_indicators.ctb tables related.
Norbert used line guide comments intended for internal use have been removed from this tables, this comments purpose only easyest collaboration work during table development and first internal review, but this inline comments are already not need, you are absolute right.
More cleaner now the tables.
If I perhaps forgot another unneed comment parts removing, feel free tell me and I doing required cosmetic cleanups.
I sent back Norbert the cleaned table files, because Norbert not have possibility to use yet the Git version control system.

Attila

Reply:
Excellent!